test/DebugInfo/NonSemantic/Shader200/FortranArray.ll

```diff
@@ -7,11 +7,11 @@
 ; RUN: llvm-dis %t.rev.bc -o - | FileCheck %s --check-prefix=CHECK-LLVM
 
 ; CHECK-SPIRV: [[#CompUnit:]] [[#]] DebugCompilationUnit
+; CHECK-SPIRV-DAG: [[#EntryFunc:]] [[#]] DebugFunction [[#]]
```

Comment:
Is is possible to modify this test to just not depend on the order? One idea is to just make all these checks as CHECK-SPIRV-DAG. Can you please try?

Thanks

Reply:
Update checks to be DAG so that order does not matter.